pkg/agent/datamodel/linux_sig_version.json

```diff
@@ -0,0 +1,4 @@
+{
+  "ostype": "LinuxSIGImageVersion",
```

Comment:
just doing this one as a first example?

Reply:
For the other versions I will be splitting them into different PRs, and since the LinuxSIGImageVersion is the one most commonly used and updated, I started with that one.